package.json

```diff
@@ -31,42 +31,43 @@
   ],
   "bugs": "https://github.com/forcedotcom/lwc-dev-mobile-core/issues",
   "dependencies": {
-    "@oclif/config": "^1.18.2",
-    "@salesforce/command": "^4.2.1",
-    "@salesforce/core": "^2.31.1",
-    "ajv": "^8.8.2",
+    "@oclif/core": "^1.20.4",
+    "@oclif/plugin-version": "1.1.3",
+    "@salesforce/command": "^5.2.24",
+    "@salesforce/core": "^3.31.19",
+    "ajv": "^8.11.2",
     "chalk": "^4.1.2",
-    "cli-ux": "^6.0.6",
     "enquirer": "^2.3.6",
-    "listr2": "^3.13.5"
+    "listr2": "^5.0.5"
   },
   "devDependencies": {
-    "@oclif/dev-cli": "^1.26.9",
-    "@types/jest": "^26.0.20",
-    "@types/node": "^16.11.12",
-    "@typescript-eslint/eslint-plugin": "^5.7.0",
-    "@typescript-eslint/parser": "^5.7.0",
-    "eslint": "^8.4.1",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-import": "^2.25.3",
-    "eslint-plugin-jsdoc": "^37.2.0",
+    "@oclif/dev-cli": "^1.26.10",
```

Comment:
Looks like, with an upgrade to `@oclif/core` (or maybe independent of it), we don't need this package anymore?

Reply:
Yes you're right, this is no longer needed.